src/mon_markings.c

```diff
@@ -0,0 +1,616 @@
+#include "global.h"
+#include "dma3.h"
+#include "graphics.h"
+#include "main.h"
+#include "menu_indicators.h"
+#include "mon_markings.h"
+#include "constants/songs.h"
+#include "sound.h"
+#include "sprite.h"
+#include "text_window.h"
+
+
+#ifdef GERMAN
```

Comment:
Pokeemerald doesn't build German--this can be removed.

Reply:
done